internal/service/identity_api/request.go

```diff
@@ -0,0 +1,31 @@
+package identity_api
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/DIMO-Network/telemetry-api/internal/config"
+	"github.com/Khan/genqlient/graphql"
+)
+
+type IdentityService interface {
+	AftermarketDevice(ctx context.Context, by AftermarketDeviceBy) (*aftermarketDeviceResponse, error)
```

Comment:
Returning a non-exported type is odd.

I know it seems silly but I would consider exposing a function that just goes device address → manufacturer token id. Really slim.